homeassistant/components/climate/nest.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, structure, device, temp_unit):
         self._unit = temp_unit
         self.structure = structure
         self.device = device
+        self._fan_list = ["On", "Auto"]
```

Comment:
Can you base these values on these constants, add the STATE_ON if necessary. https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/climate/__init__.py